<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomPage</class>
 <widget class="QWidget" name="RoomPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#RoomPage{
	background-color: rgb(39, 44, 54);
}

QFrame#frame{
	border-radius: 10px;
	background-color: rgba(225, 228, 221, 120);
}

QLineEdit{
	min-height: 45px;
	border-radius: 20px;
	background-color: #FFFFFF;
	padding-left: 20px;
	color: rgb(140, 140, 140);
}

QLineEdit:hover{
	border: 2px solid rgb(139, 142, 139);
}

QPushButton{
	min-height: 45px;
	border-radius: 20px;
	background-color: rgb(44, 66, 81);
	color: #FFFFFF;
}

QPushButton:hover{
	border: 2px solid rgb(255, 255, 255);
}

QCheckBox{
	font-size: 10px;
	color: #FFFFFF;
}

QLabel{
	color: rgb(95, 94, 108);
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="buttons_frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="add_film_btn">
        <property name="text">
         <string>Film Ekle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="invite_btn">
        <property name="text">
         <string>Davet Et</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QScrollArea" name="films_sa">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>638</width>
          <height>208</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="targets_scrollarea_2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>209</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
